frontend/src/components/account-mgmt/DuplicateUserDialog.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <AppDialog v-model="isDisplayed" title="Confirm" :isLoading="isLoading" persistent max-width="25%" @close="closeDialog">
+    <template #content>
+      <div align="center" class="confirm-dialog-text">A user with this name and email already exists. Are you sure you wish to proceed with this BCeID?</div>
+    </template>
+    <template #button>
+      <v-row justify="space-around">
+        <v-col cols="12" md="6" class="d-flex justify-center">
+          <AppButton id="dialog-continue" :primary="false" size="large" width="250px" :loading="isLoading" @click="proceed">Continue</AppButton>
+        </v-col>
+        <v-col cols="12" md="6" class="d-flex justify-center">
+          <AppButton id="dialog-cancel" size="large" width="250px" :loading="isLoading" @click="closeDialog">Cancel</AppButton>
+        </v-col>
+      </v-row>
```

Comment:
The button order is counterintuitive as we normally have Save on the right (primary) and Cancel on the left (secondary).

Reply:
Fixed.